reinforcement_learning/testnotebooks.yml

```diff
@@ -0,0 +1,7 @@
+notebooks:
+    - rl_cartpole_coach/rl_cartpole_coach_gymEnv.ipynb
+    - rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb
+
+configs:
+    rl_cartpole_coach/rl_cartpole_coach_gymEnv.ipynb: testconfig.yml
```

Comment:
Seems we will need to have a yml file for each notebook example. Since parameters such as `instance_type` and `local_mode` are universal, is there a way to have a "base" config.

Reply:
I thought about doing that, but decided against it, as it introduces additional layers of base v/s local config. Some base values may not be applicable to other notebooks. Thats why I've left only local configurations. This is simpler to manage in my opinion.

Reply:
Can you elaborate a bit on the extra maintenance effort needed, specifically, does it majorly come from the papermill side. Intuitively, the two parameters mentioned above should suffice all coach examples since we are not updating the corresponding image at this stage, also `instance_count` only applies to one or two ray examples.

Reply:
I mean there is going to be different levels of configuration introduced, which will need the developers to check both base and override configs before making any changes. This would also slightly complicate the code that runs the integration tests. So I chose to keep things simple and maintainable, at the cost of brevity.